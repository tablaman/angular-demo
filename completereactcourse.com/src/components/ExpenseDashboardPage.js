//  Expense Dashboard Page
import React from 'react'
import {
  Link
} from 'react-router-dom'
import ExpenseList from './ExpenseList';
import ExpenseListFilters from './ExpenseListFilters';

const ExpenseDashboardPage = (props) => (
  
  <div> Dashboard: expense page
    <Link to="/create">Create something</Link>
    <ExpenseListFilters />
    <ExpenseList />
  </div>
)

export default ExpenseDashboardPage;