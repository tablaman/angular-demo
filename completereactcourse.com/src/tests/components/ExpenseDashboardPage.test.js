import React from 'react';
import { shallow } from 'enzyme';
import ExpenseDashboardPage from '../../components/ExpenseDashboardPage';
import expenses from '../fixtures/expenses'

test('render ExpenseListItem with fixture data', () => {
  const wrapper = shallow( < ExpenseDashboardPage />);
  expect(wrapper).toMatchSnapshot();
});