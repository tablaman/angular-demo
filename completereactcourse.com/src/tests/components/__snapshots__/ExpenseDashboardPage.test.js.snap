// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render ExpenseListItem with fixture data 1`] = `
<div>
   Dashboard: expense page
  <Link
    replace={false}
    to="/create"
  >
    Create something
  </Link>
  <Connect(ExpenseListFilters) />
  <Connect(ExpenseList) />
</div>
`;
